stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -674,6 +736,23 @@ String getDeleteCustomerSourceUrl(@NonNull String customerId, @NonNull String so
                 "%s/%s", getAddCustomerSourceUrl(customerId), sourceId);
     }
 
+    @VisibleForTesting
+    String getPaymentMethodsUrl(@NonNull String paymentMethodId) {
```

Comment:
can you convert all these to use `getApiUrl()`?

Reply:
Converted the two other methods not using getAPIUrl to use it now